ts/Extensions/DashboardPlugins/DataGridComponent.ts

```diff
@@ -74,20 +71,64 @@ class DataGridComponent extends Component<DataGridComponent.ChartComponentEvents
      *
      * */
 
+    public static onUpdate(
+        e: KeyboardEvent,
+        store: Component.StoreTypes
+    ): void {
+        const inputElement = e.target as HTMLInputElement;
+        if (inputElement) {
+            const parentRow = inputElement
+                .closest('.hc-dg-row');
+            const cell = inputElement.closest('.hc-dg-cell');
+
+            const converter = new DataConverter();
+
+            if (
+                parentRow &&
+                parentRow instanceof HTMLElement &&
+                cell &&
+                cell instanceof HTMLElement
+            ) {
+                const dataTableRowIndex = parentRow
+                    .dataset.rowIndex;
+                const { columnName } = cell.dataset;
+
+                if (
+                    dataTableRowIndex !== void 0 &&
+                                    columnName !== void 0
```

Comment:
Some weird indentation

Reply:
Something off with my ESLint, but only when not using VSCode 🤔